composer.json

```diff
@@ -32,6 +32,7 @@
       "homepage": "https://mvorisek.cz/"
     }
   ],
+  "version": "dev-develop",
```

Comment:
This breaks composer

  [Composer\Json\JsonValidationException]
  "./composer.json" does not match the expected JSON schema:
   - version : Does not match the regex pattern ^v?\d+(((\.\d+)?\.\d+)?\.\d+)?